design_0/find_hypotenuse.py

```diff
@@ -0,0 +1,13 @@
+"""This module prompts a user to enter the lengths of two sides of a
+right triangle. Those values are then operated on in order to find the
+length of the hypotenuse. A print out of all three sides is then given.
+"""
+
+import hyp1
+
+(SIDE_A, SIDE_B) = hyp1.get_input()
```

Comment:
http://stackoverflow.com/questions/419163/what-does-if-name-main-do 

Refactor this to be importable